var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -117,6 +117,9 @@ def cmake_args(self):
             options.append('-DUSolids_DIR=%s' % spec[
                 'vecgeom'].prefix.lib.CMake.USolids)
 
+        if '+threads' in spec:
```

Comment:
is there a disable flag? if so could you also handle the ~threads case explicitly? We try to not rely on default behaviour as far as possible

Reply:
Ah, I gotcha, that makes sense. To that end, it would be beneficial to add a helper method equivalent to `AutotoolsPackage.with_or_without` -- shall I give that a shot or make something like that a separate merge request?

Reply:
OK fixed. While verifying the option was being passed correctly, I found and fixed a logic bug that always passed "CXXSTD=14" to cmake, overriding and ignoring any c++11 option.

Reply:
Feel free to add such a function, but I'd suggest a different PR for that